frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.test.tsx

```diff
@@ -42,26 +42,4 @@ describe('useFeatureSearch', () => {
         await screen.findByText(/Features:/);
         await screen.findByText(/Total:/);
     });
-
-    test('should keep only latest cache entry', async () => {
```

Comment:
what is this test (hook inside a component usage) replaced with? I only see low level hook tests but usually the most revealing tests for hooks misbehavior are usage in component tests

Reply:
Pulled this test back, but altered a little.